app/components/public/add-to-calender.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="ui list basic segment m-0 pb-0">
+<div class="ui list basic segment m-0 pb-0" style="padding-left:0px; padding-right:0px;">
```

Comment:
add pl-0 and pr-0, you have created.

Reply:
That was in a different Scss file sir, In that I had to change in one more file, That's why i didn't do this, But if you say so then I will do it now. Thanks for recommending.